oap-server/server-library/library-client/src/main/java/org/apache/skywalking/oap/server/library/client/elasticsearch/ElasticSearchClient.java

```diff
@@ -57,13 +58,15 @@
     public static final String TYPE = "type";
     private final String clusterNodes;
     private final String namespace;
+    private final String namespacePrefix;
     private final String user;
     private final String password;
     private RestHighLevelClient client;
 
     public ElasticSearchClient(String clusterNodes, String namespace, String user, String password) {
         this.clusterNodes = clusterNodes;
         this.namespace = namespace;
+        this.namespacePrefix = namespace + "_";
```

Comment:
I think this.namespacePrefix attribute is no value because we have a common method named formatIndexName.

Reply:
done